contracts/instruments/RibbonThetaVaultYearn.sol

```diff
@@ -0,0 +1,720 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.7.2;
+pragma experimental ABIEncoderV2;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import {DSMath} from "../lib/DSMath.sol";
+
+import {
+    ProtocolAdapterTypes,
+    IProtocolAdapter
+} from "../adapters/IProtocolAdapter.sol";
+import {ProtocolAdapter} from "../adapters/ProtocolAdapter.sol";
+import {IRibbonFactory} from "../interfaces/IRibbonFactory.sol";
+import {IWETH} from "../interfaces/IWETH.sol";
+import {IYearnRegistry, IYearnVault} from "../interfaces/IYearn.sol";
+import {ISwap, Types} from "../interfaces/ISwap.sol";
+import {OtokenInterface} from "../interfaces/GammaInterface.sol";
+import {OptionsVaultStorage} from "../storage/OptionsVaultStorage.sol";
+
+contract RibbonThetaVaultYearn is DSMath, OptionsVaultStorage {
```

Comment:
this name is a mouthful. yRibbonThetaVault could be cool

Reply:
I'm okay with RibbonThetaVaultYearn because it's not a user-facing product name, just the contract name